k8s-dev/cstor-pool-mgmt/replica-resource-1.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorVolumeReplica
+metadata:
+  name: pvc-ee171da3-07d5-11e8-a5be-42010a8001be-cstor-rep-9440ac
+  labels:
+#cstorpool uid will be used for reference and matching pool name
+   cstorpool.openebs.io/uid : 6eb3a8a5-707e-11e8-8a9f-54e1ad4a9dd4
+
+#this pool name is added as additional/alias name with zpool create command.
+   cstorpool.openebs.io/name: pool1-abc
+
+#pvc name is written in cvr label for reference.
+   cstorvolumereplica.openebs.io/pvc-name: demo-vol
```

Comment:
should it be PV name than pvc.. remember a PV can exist even without a PVC.
A PV which is bound to PVC1 now can be bound to a new PVC2 later.

Reply:
As discussed, this is only for reference. PV name is being used in 'Name' field. PVC name here is for reference.